apps/frontend/src/components/App.tsx

```diff
@@ -55,15 +44,16 @@ export default function App() {
           >
             <MenuIcon />
           </IconButton>
-          <Typography
-            variant="h6"
-            component="div"
-            sx={{ flexGrow: 1 }}
-            onClick={routeHome}
-          >
-            Habit Helper
+          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+            <Link
```

Comment:
Here should probably be:

```jsx
  <Link
    component={RouterLink}
    to="/"
    color="inherit"
    underline="none"
  >
    Habit Helper
  </Link>
```

In this case `Link` is the MUI Link component and not `RouterLink`. While there is not a visual difference at the moment this keeps us in the MUI component ecosystem which will make theming the app easier later.

Reply:
The new use of href does not use React Router like the above code snippet. Additionally, `style={{ textDecoration: "none", color: "inherit" }}` should be unnecessary as  `textDecoration: "none"` should be covered by using the Link components `underline="none` prop instead and `color: "inherit"` is already stated directly above it. If ``textDecoration: "none"`` is there for a reason then it should be applied using the `sx` prop from MUI.

Reply:
I tried again putting the underline and color styles in the above sx prop but they wouldn't apply to the link because CSS. The React Router was a nice catch but until we make a dedicated CSS stylesheet, if we really want to do that, this will work.